service_container/autowiring.rst

```diff
@@ -546,48 +546,105 @@ When autowiring is enabled for a service, you can *also* configure the container
 to call methods on your class when it's instantiated. For example, suppose you want
 to inject the ``logger`` service, and decide to use setter-injection::
```

Comment:
```suggestion
to inject the ``logger`` service, and decide to use setter-injection:
```

(double colon indicates the next block is a PHP code block, but that's no longer true - it's a configuration block now)